hie.yaml.stack

```diff
@@ -0,0 +1,31 @@
+cradle:
+  stack:
+    - path: "./hie-plugin-api/"
+      component: "hie-plugin-api:lib"
+
+    - path: "./test/dispatcher/"
+      component: "haskell-ide-engine:test:dispatcher-test"
+
+    - path: "./test/functional/"
+      component: "haskell-ide-engine:test:func-test"
+
+    - path: "./test/unit/"
+      component: "haskell-ide-engine:test:unit-test"
+
+    - path: "./test/plugin-dispatcher/"
+      component: "haskell-ide-engine:test:plugin-dispatcher-test"
+
+    - path: "./test/wrapper/"
+      component: "haskell-ide-engine:test:wrapper-test"
+
+    - path: "./test/utils/"
+      component: "haskell-ide-engine:lib:hie-test-utils"
```

Comment:
Does this work? Did you try this?

Reply:
Because when I do `stack repl haskell-ide-engine:lib:hie-test-utils` I get an error.

Reply:
No, that does not work, because of current lack of sublibrary support in stack. Which is why I switched to the cabal cradle.  But I put it in for completeness, in the hope that it will someday work.

Reply:
I made it work by using `--no-package-hiding`, assuming it does what we want, we may want to patch hie-bios.

Reply:
@fendor can you capture this `--no-package-hiding` possibility in an issue?